bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieClientImpl.java

```diff
@@ -267,7 +274,7 @@ public void writeLac(final BookieSocketAddress addr, final long ledgerId, final
             }
 
             toSend.release();
-        }, ledgerId);
+        }, ledgerId, conf.getUseV2WireProtocol());
```

Comment:
I don't think the value here should be `conf.getUseV2WireProtocol()`. It should be `true /* forceUseV3 *`, right?

Reply:
Yeah. will fix